draft/draft-ietf-netconf-https-notif.xml

```diff
@@ -229,7 +241,7 @@
           <figure>
             <artwork><![CDATA[GET /some/path/capabilities HTTP/1.1
 Host: example.com
-Accept: application/xml, application/json]]></artwork>
+Accept: application/xml, application/json;q=0.5]]></artwork>
```

Comment:
Should be: `Accept: application/xml;q=1.0, application/json;q=0.5`

Reply:
I was reading RFC 9110, and I thought what I read was that if q is not specified, it is preferred format.

Take this example:

Accept: text/plain; q=0.5, text/html,
       text/x-dvi; q=0.8, text/x-c
Verbally, this would be interpreted as "text/html and text/x-c are the equally preferred media types, but if they do not exist, then send the text/x-dvi representation, and if that does not exist, send the text/plain representation".[¶](https://www.rfc-editor.org/rfc/rfc9110.html#section-12.5.1-13)